### ISSUE TITLE
update RCs to Deployments

### ISSUE THREAD

**Issue by rimusz** [neshte/gluster-k8s-sidecar#3]
is any chance to update from RCs to Deployments?
the Makefile is fine, but the nodejs code I'm not really in to it.

**Comment by neshte** [comment]
yes I know, Deployments would be better... I cannot write that just now, but if you have a pull request I will happily merge it

**Comment by rimusz** [comment]
to convert templates and Makefile from RC to Deployments is not a problem for me, I not familiar with node.js
